src/utils/channel_utils.js

```diff
@@ -240,11 +251,20 @@ export function showCreateOption(config, license, channelType, isAdmin, isSystem
     return true;
 }
 
-export function showManagementOptions(config, license, channel, isAdmin, isSystemAdmin, isChannelAdmin) {
+export function showManagementOptions(state, config, license, channel, isAdmin, isSystemAdmin, isChannelAdmin) {
     if (license.IsLicensed !== 'true') {
```

Comment:
I think these license checks should only be here if we do *not* have "new permissions system".

Reply:
Now must be fine :)